pbfalcon/tusks.py

```diff
@@ -357,12 +357,12 @@ def run_scripts_in_json(input_files, output_files):
     a = json.load(open(json_fn, 'r'))
 
     writer = open(txt_fn, 'w')
-    script_dir = str(a[a.keys()[0]]['script_dir'])
-    with cd(script_dir):
-        for p_id, args in a.iteritems():
-            if 'script_fn' not in args:
-                raise ValueError("Could not find 'script_fn' in json %s key %s" % (json_fn, p_id))
-            script_fn = args['script_fn']
+    for p_id, args in a.iteritems():
+        if 'script_fn' not in args:
+            raise ValueError("Could not find 'script_fn' in json %s key %s" % (json_fn, p_id))
```

Comment:
It is very helpful to someone reading the message if you use `%r` instead of `%s` so we can see quotation marks. That is very important for empty strings and still helpful for non-empty strings.